themes/spichka/resources/styles/front-page.scss

```diff
@@ -0,0 +1,74 @@
+.section-banner-video {
+  height: 100vh;
+  align-items: center;
+  display: flex;
+
+  .banner-video {
+    position: absolute;
+    inset: 0;
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    filter: brightness(60%);
+  }
+
+  .header {
+    color: white;
+    position: relative;
```

Comment:
position is not used now. Flexboxes can do all the job. We can leave a TODO here for future